File: Solutions/2534/csharp/Program.cs
using System;
using System.Collections.Generic;
using System.Linq;

namespace Quera {
    public static class Program {
        public static void Main() {
            var n = GetInputs<int>(1).Single();
            var columns = GetInputs<int>(n);

            var average = columns.Average();
            var result = columns.Where(column => column > average)
                .Sum(column => column - average);

            Console.WriteLine(result);
        }

        private static List<T> GetInputs<T>(int count) {
            var result = new List<T>(count);
            for (var i = 0; i < count; i++) {
                result.Add((T) Convert.ChangeType(Console.ReadLine(), typeof(T)));
            }

            return result;
        }
    }
}

File: Solutions/2534/python3/program.py
from typing import List


def get_inputs(num_of_lines: int) -> List[str]:
    result = []
    for i in range(num_of_lines):
        result.append(input())
    return result


# Sample
num_of_coin_columns = int(input())
coin_columns = list(map(int, get_inputs(num_of_coin_columns)))

average = int(sum(coin_columns) / num_of_coin_columns)

minutes = 0
for coin_column in coin_columns:
    if coin_column >= average:
        continue
    minutes += average - coin_column

print(minutes)
